help/develop/how-to/test-packages.md

```diff
@@ -97,55 +97,56 @@ This package contains a collection of test cases that can serve as illustrative
+    
+    Note that publish to the NPM registry is executed for the master branch only.
+3. Check that the GitHub Actions workflow finished successfully.
+4. The results are available in the actions output.
 
 ### Troubleshooting failed tests in GitHub Actions
 
-When a GitHub Action completes, you can access the "Artifacts" section on its main page ("Summary"), which contains a zip archive with the results of the tests for that specific action. This archive typically includes three important files that can help you diagnose and resolve the problems:
+When a GitHub Action completes, you can access the `Artifacts` section on its main page (Summary), which contains a zip archive with the results of the tests for that specific action. This archive includes three important files that can help you diagnose and resolve the problems:
 
-1. `test-console-output.log`: This file contains the console log records generated during   the test execution. The console output often includes valuable information such as error messages, stack traces, and debugging statements. It's a good starting point for understanding what went wrong during the test run.
+1. `test-console-output.log` contains the console log records generated during the test execution, including error messages, stack traces, and debugging statements. It's a good starting point for understanding what went wrong during the test run.
 
-2. `test-record.mp4`: This file is a video recording of the test execution. The video recording can be immensely helpful in scenarios where tests are failing intermittently or in cases where the issue is related to the user interface or visual components of your application. Watching the test execution can provide insights into the behavior of the application and identify potential areas of concern.
+2. `test-record.mp4` is a video recording of the test execution, which can be immensely helpful in scenarios where tests are failing occasionally or in cases where the issue is related to the user interface or some visual components.
 
-3. `test-report.csv`: This file is a tabular representation of the test results. It contains information about which tests passed, which failed, and also additional details such as test duration or errors. The test report can give you a quick overview of the overall test status and pinpoint specific failing tests.
+3. `test-report.csv` is a tabular representation of the test results. It contains information about which tests passed, which failed, and also additional details such as test duration or errors.
 
 ## Test manager
 
-'Test manager' is a tool within the Datagrok platform that provides a convenient interface to select and run package
-unit tests with further results exploration.
-'Test manager' itself is a part of the DevTools package.
+Test Manager is a tool within the Datagrok platform that provides a convenient interface to select and run package tests with further results exploration.
```

Comment:
I think it's obvious that Test Manager is the Datagrok tool?:)
I would simplify this sentence like this, but it's debatable.
> Test Manager is a tool used to run package tests and view their results.

Reply:
Cut even more: [Test Manager](link to public instance) lets you run package tests and review test results.